src/components/organisms/about-content/about-content.tsx

```diff
@@ -1,38 +1,125 @@
+          <DefaultText className={"text-dark text-[16px]"}>Follow us on Social Media:</DefaultText>
+          <Flex className="flex gap-12 w-full items-center justify-center">      
+            <Icon Icon={SiTwitter} iconAlt="Twitter for Tech Is Hiring" url="https://www.twitter.com/techishiring" className={"text-primary"}/>
+            <Icon Icon={SiLinkedin} iconAlt="LinkedIn for Tech Is Hiring" url="https://www.linkedin.com/company/techishiring" className={""} />
+            <Icon Icon={SiFacebook} iconAlt="LinkedIn for Tech Is Hiring" url="https://www.linkedin.com/company/techishiring" className={"text-primary"}/>
+          </Flex>
+        </Box>
+      </Box>
+      <article className={"flex justify-center text-center md:text-left text-lg md:text-xl leading-[41px] py-0 md:py-20 items-center flex-col gap-8 font-700 text-altDark "}>
+        <DefaultText as={"p"} w={{base:"80%", md:"70%"}}>
+          <b>TechIsHiring</b> tries to assist tech professionals in finding employment 
+          by encouraging meaningful interactions between people looking for employment 
+          and those who can assist, rather than focusing primarily on advertising job openings.
+        </DefaultText>
+        <DefaultText as={"p"} w={{base:"80%", md:"70%"}}>
+          During the epidemic, <b>Chad Stewart</b>, the <b>founder of TechIsHiring</b>, noticed that many 
+          people were tweeting about job openings they had discovered and advertising possibilities 
+          that they had come across, which is how <b>TechIsHiring</b> came to be. Since there was no one place  
+          where these tweets could be collected, the hashtag <Link
+            url={"https://twitter.com"}>
+            <b className="text-primary font-800 cursor-pointer">
+               #TechIsHiring
+            </b>
+          </Link> and the TechIsHiring account were created.
+        </DefaultText>
+        <Box display={{base: "flex", md: "none"}} w={"80%"} flexDirection={"column"}>
+          <Divider orientation="horizontal" borderColor={"#838383"}/>
+          <DefaultText className={"text-black text-[23px] capitalize pt-14 pb-8 font-bold "}>Follow us</DefaultText>
+          <Flex className="flex gap-12 w-full items-center justify-center">      
+            <Icon Icon={SiTwitter} iconAlt="Twitter for Tech Is Hiring" url="https://www.twitter.com/techishiring" className={"text-primary"}/>
+            <Icon Icon={SiLinkedin} iconAlt="LinkedIn for Tech Is Hiring" url="https://www.linkedin.com/company/techishiring" className={""} />
+            <Icon Icon={SiFacebook} iconAlt="LinkedIn for Tech Is Hiring" url="https://www.linkedin.com/company/techishiring" className={"text-primary"}/>
+          </Flex>
+        </Box>
+        <article className="text-primary py-8 px-10 md:px-0">‘Where great Jobs and great Engineers come to find one another!’</article>
+        
+      </article>
+
+    </Box>
```

Comment:
Just noticed this and wanted to ask why you chose to use the Chakra UI components specifically to build out the layout for this page rather than just using Tailwind?

Reply:
I prefer using chakra for now 

I also use tailwind but the project also allows chakra that was why I used it 

Please is anything wrong here?

Reply:
So the reason why I ask is cause most of the layout for the project is defined with Tailwind. Chakra UI components tend to be used on specific components or some complex components I didn't want to build myself. Honestly I did a poor job of communicating that which is something I need to rectify.

One of the reasons this is important is because there's also a discussion about whether we might move away from Chakra UI in the future in issue #60. If we were to decide to move to another UI library, we'd effectively have to redo this page.

Is it possible you could use Tailwind for the layout specifically? It would help keep the project consistent.

Reply:
> So the reason why I ask is cause most of the layout for the project is defined with Tailwind. Chakra UI components tend to be used on specific components or some complex components I didn't want to build myself. Honestly I did a poor job of communicating that which is something I need to rectify.
> 
> One of the reasons this is important is because there's also a discussion about whether we might move away from Chakra UI in the future in issue #60. If we were to decide to move to another UI library, we'd effectively have to redo this page.
> 
> Is it possible you could use Tailwind for the layout specifically? It would help keep the project consistent.

Sure no prob I'll fix this once my power supply has been restored 

I think this should be added to the contributing.md or readme for new contributors

Reply:
I'll make an issue on that in a moment. Thanks for pointing that out.